app/services/flow/flow_state_machine.rb

```diff
@@ -25,6 +25,15 @@ def update
         increment_step_name_counts
         analytics.track_event(analytics_submitted, result.to_h.merge(analytics_properties))
       end
+
+      if irs_attempts_api_session_id &&
```

Comment:
Dont like this initial implementation, wanted a way to properly retrieve the relevant data, but also wouldn't blow up existing steps that work/input htis data in other ways.